webrtc.html

```diff
@@ -3207,7 +3215,7 @@ <h3>
   constructor(optional RTCConfiguration configuration = {});
   Promise&lt;RTCSessionDescriptionInit&gt; createOffer(optional RTCOfferOptions options = {});
   Promise&lt;RTCSessionDescriptionInit&gt; createAnswer(optional RTCAnswerOptions options = {});
-  Promise&lt;undefined&gt; setLocalDescription(optional RTCLocalSessionDescriptionInit description = {});
+  Promise&lt;undefined&gt; setLocalDescription(optional OptionalRTCSessionDescriptionInit description = {});
```

Comment:
dictionary names are not exposed to JS developers, but they do take a slot in the IDL space - I'd be reluctant to make such a change at this stage.

If we we were to change it, I think it would need to be `RTCOptionalSessionDescriptionInit` (since pretty much all of our IDL names are prefixed with RTC)

Reply:
I don't have a strong preference here, but since the name doesn't matter much and it's only used for the SLD call, I suggest we keep the old name and merge this editorial PR.